python/lsst/afw/display/interface.py

```diff
@@ -147,11 +147,24 @@ def __init__(self, frame=None, backend=None, *args, **kwargs):
                     setDefaultBackend("ds9")
                 except RuntimeError:
                     setDefaultBackend("virtualDevice")
+            else: # a default was set, merge optional arguments
+                try:
+                    self._args += args
+                except AttributeError:
+                    self._args = args
+                try:
+                    self._kwargs.update(kwargs)
+                except AttributeError:
+                    self._kwargs = kwargs
 
             backend = Display._defaultBackend
+        else: # a backend was specified, discard default optional arguments
+            self._args = args
+            self._kwargs = kwargs
 
         self.frame = frame
-        self._impl = _makeDisplayImpl(self, backend, *args, **kwargs)
+        
```

Comment:
I didn't notice the whitespace -- I can take it out. 

I'm using Sublime Text 3 -- I wonder if there's a way to configure it to point out issues like this? Or would it be covered by `pep8`?